provision/warewulf-provision.spec.in

```diff
@@ -1,171 +1,105 @@
 %{!?_rel:%{expand:%%global _rel 0.%(test "@GITVERSION@" != "0000" && echo "@GITVERSION@" || git show -s --pretty=format:%h || echo 0000)}}
 %define debug_package %{nil}
-%define _cross_compile 0%{?cross_compile}
+%{!?wwsrvdir:%{expand:%%define wwsrvdir %{_localstatedir}/warewulf}}
 
 Name: warewulf-provision
-Summary: Warewulf - Provisioning Module
+Summary: Warewulf - System provisioning core
 Version: @PACKAGE_VERSION@
 Release: %{_rel}%{?dist}
-#Release: 1.%{?_dist}
 License: US Dept. of Energy (BSD-like) and BSD-3 Clause
-Group: System Environment/Clustering
 Source: %{name}-%{version}.tar.gz
 ExclusiveOS: linux
-Requires: warewulf-common %{name}-initramfs-%{_arch} = %{version}-%{release}
+Requires: warewulf-common
+Requires: %{name}-initramfs-%{_arch} = %{version}-%{release}
 Conflicts: warewulf < 3
 BuildRequires: warewulf-common
-BuildRequires: libselinux-devel libacl-devel libattr-devel
-BuildRequires: libuuid-devel device-mapper-devel xz-devel
+BuildRequires: libselinux-devel, libacl-devel, libattr-devel
+BuildRequires: libuuid-devel, device-mapper-devel, xz-devel
 BuildRequires: libtirpc-devel
 
-%if 0%{?_cross_compile}
-%global CONF_CROSS --enable-cross-compile %{CONF_FLAGS}
+%if 0%{?rhel:1}
+%global httpsvc httpd
+%global httpgrp apache
+%global tftpsvc tftp-server
+%if %{rhel} >= 8
+BuildRequires: systemd
+%global dhcpsrv dhcp-server
+%else # rhel < 8
+%global dhcpsrv dhcp
+%endif # rhel 8
+%else # sle_version
+BuildRequires: systemd-rpm-macros
+%global httpsvc apache2
+%global httpgrp www
+%global tftpsvc tftp
+%global dhcpsrv dhcp-server
+%endif # rhel
+
+# If multiple architectures are needed, set
+# --define="cross_compile 1" as an rpmbuild option
+%if 0%{?cross_compile}
+%global CROSS_FLAG --enable-cross-compile
 %if "%{_arch}" == "x86_64"
 BuildRequires: gcc-aarch64-linux-gnu
 %endif # x86_64
 %if "%{_arch}" == "aarch64"
 BuildRequires: gcc-x86_64-linux-gnu
 %endif # aarch64
+%else
+%undefine CROSS_FLAG
 %endif # cross_compile
 
+# New RHEL and SLE include the required FS tools
 %if 0%{?rhel} >= 8 || 0%{?sle_version} >= 150000
-BuildRequires: parted e2fsprogs libarchive bsdtar
-Requires: parted autofs e2fsprogs libarchive bsdtar
+%global localtools 1
+BuildRequires: parted, e2fsprogs
+Requires: parted, autofs, e2fsprogs
+BuildRequires: libarchive.so.13()(64bit)
+Requires: libarchive.so.13()(64bit)
 %global CONF_FLAGS --with-local-e2fsprogs --with-local-libarchive --with-local-parted --with-local-partprobe
+%else
+%global localtools 0
+Requires: %{name}-gpl_sources = %{version}-%{release}
+Provides: parted = 3.2
```

Comment:
Same thing, should we advertise to the system that we're providing parted, e2fsprogs, and libarchive?